internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -574,3 +574,32 @@ func convertCommand(command *manifest.CommandOverride) ([]string, error) {
 	}
 	return out, nil
 }
+
+func convertPublish(p *manifest.PublishConfig) (*template.PublishOpts, error) {
+	if p == nil || p.Topics == nil {
+		return nil, nil
+	}
+	publishers := template.PublishOpts{}
+	// convert the topics to template Topics
+	for _, topic := range p.Topics {
+
+		err := validateName(topic.Name)
+		if err != nil {
+			return nil, err
+		}
+
+		workers, err := validateWorkers(topic.AllowedWorkers)
```

Comment:
What do you think we combine `validateName` and `validateWorkers` to be `validateTopic`?

Reply:
Combining these is difficult due to workers being individually validated with validate name and then converted, so I separated out a convertTopic function instead to handle it.